src/Driver.cpp

```diff
@@ -40,6 +45,10 @@ class SPFConsumer : public ASTConsumer {
 public:
   explicit SPFConsumer(llvm::StringRef fileName) : fileName(fileName.str()) {}
   void HandleTranslationUnit(ASTContext &Ctx) override {
+    if (EntryPoint.empty()) {
+      llvm::errs() << "\033[31m--entry-point flag must be specified (-h for usage)\033[0m\n";
```

Comment:
what systems will this work properly on? (I assume that's a color specification)

Reply:
It is.

From [wikipedia](https://en.wikipedia.org/wiki/ANSI_escape_code#Platform_support):

> [Unix-like systems] Terminal emulators for communicating with local programs as well as remote machines and the text system console almost always support ANSI escape codes. This includes terminal emulators such as xterm, rxvt, GNOME Terminal, and Konsole on systems with X11-based or Wayland-based window systems, and Terminal.app and third-party terminal emulators such as iTerm2 on macOS.

> [Windows] In 2016, Microsoft released the Windows 10 version 1511 update which unexpectedly implemented support for ANSI escape sequences, over two decades after the debut of Windows NT.[13] This was done alongside Windows Subsystem for Linux, allowing Unix-like terminal-based software to use the sequences in Windows Console. Unfortunately this defaults to off, but Windows PowerShell 5.1 enabled it. PowerShell 6 made it possible to embed the necessary ESC character into a string with `e.[14] Windows Terminal, introduced in 2019, supports the sequences by default, and Microsoft intends to replace the Windows Console with Windows Terminal.[15]


Judging by that I would say essentially everywhere :shrug: .

Reply:
Ok cool, I just remembered from a few years back when I tried to do something like this it was very system specific (maybe I used a different method). But this seems universal enough, and it works on my zsh and presumably your bash so :+1: